### PR TITLE
Basic guidance for deploying an ASP.Net Core Web App to Azure App Service

### LINKED ISSUE #11997: Provide guidance for publishing .NET Core Web App to an Azure App Service from Visual Studio code
Visual Studio code is one of the favorite IDE recently. ASP.Net Core as a framework as well.

It would be nice to have guidance on how to do that to reach those people who has this preference combination.

I'll working on creating something.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 71a44c72-d256-fbee-ea70-2eb7472f8182
* Version Independent ID: 157027ff-0bad-828d-a442-a98acd219fba
* Content: [Publish an ASP.NET Core app to Azure with Visual Studio](https://docs.microsoft.com/en-us/aspnet/core/tutorials/publish-to-azure-webapp-using-vs?view=aspnetcore-2.2)
* Content Source: [aspnetcore/tutorials/publish-to-azure-webapp-using-vs.md](https://github.com/aspnet/Docs/blob/master/aspnetcore/tutorials/publish-to-azure-webapp-using-vs.md)
* Product: **aspnet-core**
* Technology: **aspnetcore-tutorials**
* GitHub Login: @Rick-Anderson
* Microsoft Alias: **riande**

### PR DESCRIPTION
**Pull request by ricardoserradas**
Fixes #11997

Here's the guidance for publishing an ASP.Net Core Web App from Visual Studio Code, leveraging on the Azure App Service VS Code extension.

### CODE REVIEW

File: aspnetcore/tutorials/publish-to-azure-webapp-using-vscode.md [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+
+With this tutorial, you'll learn how to create an ASP.Net Core MVC Application
+and deploy it within Visual Studio Code.
+
+## Set up
+
+- Open a [free Azure account](https://azure.microsoft.com/free/dotnet/) if you don't have one.
+- Install [.NET Core](https://dotnet.microsoft.com/download)
+- Install [Visual Studio Code](https://code.visualstudio.com/Download)
+  - Instal the [Azure App Service Extension](https://marketplace.visualstudio.com/items?itemName=ms-azuretools.vscode-azureappservice) to Visual Studio Code and configure it before proceeding
```
Comment (metoly) [inline aspnetcore/tutorials/publish-to-azure-webapp-using-vscode.md]: You also need to install the C# extension to build and compile .NET Core projects. Can you please add guidance to do so before installing the "Azure App Service Extension".
Something along the lines of:
- Install the [C# Extension] (https://marketplace.visualstudio.com/items?itemName=ms-vscode.csharp) to Visual Studio Code.

File: aspnetcore/tutorials/publish-to-azure-webapp-using-vscode.md [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+
+## Intro
+
+With this tutorial, you'll learn how to create an ASP.Net Core MVC Application
+and deploy it within Visual Studio Code.
+
+## Set up
+
+- Open a [free Azure account](https://azure.microsoft.com/free/dotnet/) if you don't have one.
+- Install [.NET Core](https://dotnet.microsoft.com/download)
```
Comment (metoly) [inline aspnetcore/tutorials/publish-to-azure-webapp-using-vscode.md]: The user needs to install the SDK, not the runtime. Can we be more specific about this?

File: aspnetcore/tutorials/publish-to-azure-webapp-using-vscode.md (hunk 1 above)
Comment (ricardoserradas) [reply aspnetcore/tutorials/publish-to-azure-webapp-using-vscode.md]: Done. Your suggestion was used here, thanks!

File: aspnetcore/tutorials/publish-to-azure-webapp-using-vscode.md (hunk 2 above)
Comment (ricardoserradas) [reply aspnetcore/tutorials/publish-to-azure-webapp-using-vscode.md]: The best way I found to be more specific here was to explicitly say "SDK" on the link text.

There's no link for a page for downloading the SDK specifically.

**Pull request merged**